### PR TITLE
When we listen to any audio message then it will be good if the time is displayed

### LINKED ISSUE #1557: When we listen to any audio message then it will be good if the timer is displayed.
when we listen to any audio message then the timer should start until the audio ends so that we can see how much audio we have listened

### PR DESCRIPTION
**Pull request by devyaniChoubey**
…

@RocketChat/ReactNative

Closes #1557 

Peek 2020-01-14 01-36

### CODE REVIEW

File: app/containers/message/Audio.js [hunk 1]
```diff
@@ -197,7 +197,7 @@ class Audio extends React.Component {
 						onValueChange={this.onValueChange}
 						thumbImage={isIOS && { uri: 'audio_thumb', scale: Dimensions.get('window').scale }}
 					/>
-					<Text style={[styles.duration, { color: themes[theme].auxiliaryText }]}>{this.duration}</Text>
+					{paused ? <Text style={[styles.duration, { color: themes[theme].auxiliaryText }]}>{this.duration}</Text> : <Text style={[styles.duration, { color: themes[theme].auxiliaryText }]}>{formatTime(currentTime)}</Text>}
```
Comment (diegolmello) [inline app/containers/message/Audio.js]: The whole `<Text style={[styles.duration, { color: themes[theme].auxiliaryText }]}>...</Text>` is repeated twice.
Why not creating a new variable and use it like this?
```
<Text style={[styles.duration, { color: themes[theme].auxiliaryText }]}>
    {newVariable}
</Text>
```

**Comment by devyaniChoubey** [comment]
@diegolmello sorry sir for late reply I was busy in my college exams.
I will do it

**Comment by devyaniChoubey** [comment]
@sarthakpranesh  I will try to do the changes today only.Please let me do it. I was busy in my college exams.

**Comment by djorkaeffalexandre** [comment]
Refer to #2073.

**Pull request closed without merging**